docs/configuration/configuration.md

```diff
@@ -577,6 +577,12 @@ role: <string>
 # tasks and services that don't have published ports.
 [ port: <int> | default = 80 ]
 
+# Optional filters to limit the discovery process to a subset of available
```

Comment:
The usual paragraph about only using this for performance reasons should be added.

Reply:
This comment is still unaddressed.